src/polymerday-sponsors-list.html

```diff
@@ -36,13 +76,31 @@
       is: 'polymerday-sponsors-list',
 
       properties: {
```

Comment:
I guess we need this properties:

- sponsors (Array)
- header (String)
- logos-size (String): Enumerable with option like XL, L, M, S or something like this.